### DISCUSSION TITLE
In docker The Chinese characters in the Cypress console log show garbled code

### DISCUSSION THREAD

**Discussion by Silence-dream** [cypress-io/cypress#27153] [Questions and Help] [answered]
cypress: 12.16.0
dokcer : cypress/included:latest
chrome: 114
How can I solve this problem?

![cypress-screenshot](https://github.com/cypress-io/cypress/assets/42824008/358878d7-64b5-41dc-959d-d983d846c1b1)

https://github.com/cypress-io/cypress/assets/42824008/1e7a91fb-93c0-4265-a5d0-235c27fe4175

**Comment by idedoncker** [comment, marked as answer]
cypress: 12.17.1
docker : cypress/included:12.17.1
electron: 106

We had troubles in our visual regression testing where Chinese characters were rendered as rectangles similar to your console log result. We resolved it by installing the [fonts-wqy-zenhei](https://packages.debian.org/sid/fonts-wqy-zenhei) font in our Dockerfile.

**Reply by Jerry0031** [reply]
May I ask how to install this font in Dockerfile? I try apt-get this font, or copy mac PingFang to Docker but all useless

**Comment by LinmonK** [comment]
As mentioned earlier, I tried to solve this problem by installing fonts. As Cypress/included uses Debian and the network is not very smooth in China, I first cleared the default apt download source, added the domestic Alibaba Cloud image source, and finally downloaded it. The relevant Dockerfile is as follows:

``` Dockerfile
# 删除默认镜像源
RUN rm -rf /etc/apt/sources.list.d

# 添加多个国内镜像源
RUN echo 'deb http://mirrors.aliyun.com/debian/ bookworm main non-free contrib' > /etc/apt/sources.list && \
    echo 'deb-src http://mirrors.aliyun.com/debian/ bookworm main non-free contrib' >> /etc/apt/sources.list && \
    echo 'deb http://mirrors.aliyun.com/debian-security bookworm-security main' >> /etc/apt/sources.list && \
    echo 'deb http://mirrors.aliyun.com/debian/ bookworm-updates main non-free contrib' >> /etc/apt/sources.list && \
    echo 'deb http://mirrors.aliyun.com/debian/ bookworm-backports main non-free contrib' >> /etc/apt/sources.list 

#更新包列表并安装字体
RUN apt-get update && \
    apt-get install -y fonts-wqy-microhei fonts-wqy-zenhei && \
    rm -rf /var/lib/apt/lists/*

```
